modules/DNN/NeuronLayerMat.h

```diff
@@ -28,6 +29,7 @@ namespace DirectGraphicalModels {
 			DllExport void	setValues(const Mat& values);
 			DllExport Mat   getValues(void) const { return m_values; }
 			DllExport int   getNumNeurons(void) const { return m_values.rows; }
+            DllExport Mat   getWeights(void) const { return m_weights; }
```

Comment:
Please roll back all the changes in this file

Reply:
done